packages/api/lambdas/create-reconciliation-report.js

```diff
@@ -497,20 +504,59 @@ async function createReconciliationReport(params) {
     Bucket: systemBucket,
     Key: reportKey,
     Body: JSON.stringify(report)
-  }).promise()
-    .then(() => null);
+  }).promise();
+}
+
+/**
+ * start the report generation process and save the record to database
+ * @param {Object} params - params
+ * @param {string} params.systemBucket - the name of the CUMULUS system bucket
+ * @param {string} params.stackName - the name of the CUMULUS stack
+ *   DynamoDB
+ * @returns {Promise<null>} a Promise that resolves when the report has been
+ *   uploaded to S3
+ */
+async function processRequest(params) {
+  const { systemBucket, stackName } = params;
+  const reportStartTime = moment.utc();
+  const reportRecordName = `inventoryReport-${reportStartTime.format('YYYYMMDDTHHmmssSSS')}`;
+  const reportKey = s3ReportKey(stackName, reportStartTime);
```

Comment:
Should we pass `reportRecordName` into `s3ReportKey` instead of the `reportStartTime`?